ert_gui/tools/plot/plot_api.py

```diff
@@ -174,27 +161,22 @@ def observations_for_key(self, case_name, key):
         case = self._get_case(case_name)
 
         with Storage.session() as client:
+            response = client.get(f"/ensembles/{case['id']}/records/{key}/observations")
+            self._check_response(response)
+
+            obs = response.json()[0]
```

Comment:
Could this throw an exception? Perhaps put in try

Reply:
Yes, it could, but if there is an exception thrown it means the observation response schema was changed and that error would be caught by the API compatibility tests

Reply:
Thought a bit more and realized that I should not blindly trust in our very good testing suite and if we ever get in the situation where a schema change snicks through without notice might not be nice to crash the application. Will add a try catch.